### PR TITLE
Update DESCRIPTION

### PR DESCRIPTION
**Pull request by shajoezhu**
# Pull Request

### CODE REVIEW

**Comment by shajoezhu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by shajoezhu** [comment]
I have read the CLA Document and I hereby sign the CLA

File: .github/workflows/scheduled.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ jobs:
       strategy: ${{ matrix.test-strategy }}
       additional-env-vars: |
         PKG_SYSREQS_DRY_RUN=true
+      extra-deps: |
+        jsonlite (>= 1.8.8)
```
Comment (shajoezhu) [inline .github/workflows/scheduled.yaml:25]:
```suggestion
        jsonlite (>= 1.8.7)  
```

**Comment by shajoezhu** [comment]
@pawelru , i think we are good to go

**Review by pawelru — APPROVED** [review]
approved based on https://github.com/insightsengineering/nestcolor/actions/runs/10455640884

Thanks!

**Pull request merged**